src/main/java/core/basesyntax/dao/CsvFileReader.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.dao;
```

Comment:
move all interfaces and classes from dao package to service, they're not related to DAO layer at all, you don't work with DB there, the same for test classes for them